waku/v2/node/peer_manager/peer_manager.nim

```diff
@@ -233,59 +248,31 @@ proc dialPeer*(pm: PeerManager, remotePeerInfo: RemotePeerInfo, proto: string, d
     trace "Adding newly dialed peer to manager", peerId= $remotePeerInfo.peerId, address= $remotePeerInfo.addrs[0], proto= proto
     pm.addPeer(remotePeerInfo, proto)
```

Comment:
Perhaps for the next PR, but is this still necessary if dialing adds the peer in the underlying peer store in any case?

Reply:
Yes, good point.

I'm actually preparing a PR where I will be decoupling **dialing** from **adding**. So just discv5,px,rpc,bootrap will be the ones calling `addPeer`. So dial peer won't be adding peers anymore.